src/config.ts

```diff
@@ -100,7 +100,7 @@ export async function loadConfig(file?: string, trace?: boolean): Promise<Saddle
 
   return {
     ...merged,
-    ...trace === undefined ? {} : { trace }
+    ...trace === true ? { trace } : {}
```

Comment:
This is bc it was passing `false` by default? Also `trace ? {trace} : {}` or `{trace: trace}`

Reply:
Well, I don't want to test for "truthy" values-- I want to say: if you've set it to true, then use it, otherwise, don't override. This is because there's a lot of places where we check and say: `useTrace = !!process.env['trace']` or such-- and really that should be: `useTrace = !!process.env['trace'] ? true : undefined` but that behavior isn't super nice either. Then at least, this would be: `...trace !== undefined ? { trace } : {}`. Overall, there's just a little bad design on whether functions are accepting "use this trace value" or "possibly include a default trace value"